week1/assignment1/index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>가영쓰 미국여행기</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="section2.css" />
+    <link rel="stylesheet" href="section3.css" />
+    <link rel="stylesheet" href="section4.css" />
+  </head>
+  <body>
+    <header id="main1">
+      <h3>가영쓰 미국여행기😋</h3>
+    </header>
+    <section class="preview">
+      <img src="img/img1.jpg" alt="미국여행사진" />
+      <img src="img/img2.jpg" alt="미국여행사진" />
+      <img src="img/img3.jpg" alt="미국여행사진" />
+      <img src="img/img4.jpg" alt="미국여행사진" />
+      <img src="img/img5.jpg" alt="미국여행사진" />
+      <img src="img/img6.jpg" alt="미국여행사진" />
+      <img src="img/img7.jpg" alt="미국여행사진" />
+      <img src="img/img8.jpg" alt="미국여행사진" />
+      <img src="img/img9.jpg" alt="미국여행사진" />
+      <img src="img/img10.jpg" alt="미국여행사진" />
+      <img src="img/img11.jpg" alt="미국여행사진" />
+      <img src="img/img12.jpg" alt="미국여행사진" />
+    </section>
+
+    <nav>
+      <span class="categories">
+        <h3>🌠클릭하면 이동해요!🌠</h3>
+      </span>
+      <div class="city">
+        <a href="#main2"><b>Las Vegas</b></a>
+        <a href="#main3"><b>San diego</b></a>
+        <a href="#main4"><b>Los Angeles</b></a>
+      </div>
```

Comment:
```suggestion
<ul>
    <li><a href=" ~~></a></li>
```
이런식으로 ul-li태그를 써봐도 좋을 것같아요!